src/compiler/utilities.ts

```diff
@@ -1586,35 +1586,28 @@ namespace ts {
             ((node as JSDocFunctionType).parameters[0].name as Identifier).escapedText === "new";
     }
 
-    export function getAllJSDocs(node: Node): (JSDoc | JSDocTag)[] {

```

Comment:
this was internal, right? I don't see a jsdoc on it, so I assume so.

Reply:
API tests should ensure we don't delete internal things.